src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3139,24 +3142,24 @@ private ReadOnlyBindingDiagnostic<AssemblySymbol> GetSourceDeclarationDiagnostic
             }
 
             // NOTE: Concatenate the CLS diagnostics *after* filtering by tree/span, because they're already filtered.
-            ReadOnlyBindingDiagnostic<AssemblySymbol> clsDiagnostics = GetClsComplianceDiagnostics(syntaxTree, filterSpanWithinTree, cancellationToken);
+            ReadOnlyBindingDiagnostic<AssemblySymbol> clsDiagnostics = GetClsComplianceDiagnostics(syntaxTree, filterSpanWithinTree, symbolFilter, cancellationToken);
```

Comment:
>GetClsComplianceDiagnostics [](http://example.com/codeflow?start=71&length=27)

Perhaps orthogonal, but we should skip CLS compliance checking entirely during EnC compilation.